core/src/main/resources/hudson/tasks/Shell/help-unstableReturn.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    If set, the shell exit code that will be interpreted as an unstable build result. On Unix-like systems the value between 0-255.
```

Comment:
Second sentence missing its verb.

Also, seems overly terse, could use an explanation what this means. E.g. not aborting the build if that's the result, but continuing.